publisher/sample-publisher/integration/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG EMPIRIC_PUBLISHER_BASE_IMAGE_TAG=0.8.0
```

Comment:
Need to add this file to empiric-package/.bumpversion.cfg so that we bump this version on upgrade.